src/plugins/visualizations/public/wizard/new_vis_modal.tsx

```diff
@@ -185,4 +199,5 @@ class NewVisModal extends React.Component<TypeSelectionProps, TypeSelectionState
   }
 }
 
-export { NewVisModal };
+// eslint-disable-next-line import/no-default-export
```

Comment:
Could you add another comment please above, that the default export is needed to work with React.lazy, to clarify why we disable linting here.